---
title: Aassociate members and partners
description: Aassociate members and partners
showHeader: false

---

## Aassociate members and partners

{{< pinkbox >}}
Human Pangenome Reference Consortium is an international effort focused on developing an inclusive collection of human reference genomes that represent human haplotype diversity and the tool ecosystem needed to fully utilize this resource. The consortium welcomes the participation of all additional academic, industry partners and non-profit organizations. The value for these members is a highly interactive research environment that involves many aspects of the Human Pangenome Reference Research.
{{< /pinkbox >}}

{{< br >}}

<div style="background-color: #e7ecffbf; padding: 1rem; font-size: 1.2rem">

**Data Availability** – All data contributed to or generated by the HPRC will be made publicly available for research purposes.

**Data Use** - All data contributed to or generated by the HPRC will be available for general research use.

**Analysis and Assembly** - Contributions to literature curation, disease/gene association efforts, variant level pathogenicity determinations, or other activities undertaken by collaborators are encouraged. All results will be made freely and publicly available.

**Non-endorsement** – Collaborating with HPRC does not imply endorsement of the product or the collaborator by the NIH.

**Non-exclusivity** – Collaborators will not have exclusive agreements with the HPRC project for any activity they undertake as part of their involvement in the project.

**Publication** – All collaborators are expected to abide by the HPRC publication policy.

**Information Form** – All HPRC collaborators must complete the HPRC Associate Membership Information Form and return it to the HPRC Coordinating Center. Our purpose in collecting this information is help distribute information to HPRC members and also to promote cross channel communications by providing a list of associate members on the HPRC website.
</div>

{{< br >}}

<div id="application_form_div">
    The policy for associate members can be accessed <a
        id="policy_form"
        target="_blank"
        href="HPRCAssociateMembersPolicy.pdf">here</a
    >. HPRC associate membership application is available <a
        id="application_form"
        target="_blank"
        href="HPRCCollaborationInformationFormV2.pdf">here</a
    >.
</div>

{{< columns count=2 >}}
{{< column >}}

### Current Associate Members

| **Name** | **Affiliation** | **Start Date** |
| --- | --- | --- |
| Hardip Patel | Austrailian National University | September 2020 |
| Simon Easteal | Australian National University| September 2020 |
| Steven Salzberg | Johns Hopkins University | September 2020 |
| Jeffrey Rosenfeld | Rutgers University | December 2020 |
| George Liu | USDA ARS | March 2021 |
| Obed Garcia | Stanford University | June 2021 |
| Omar Eduardo Cornejo Ordaz | Washington State University | June 2021 |
| Shilpa Garg | University of Copenhagen | August 2021 |
| Ahmad Abou Tayoun | Al Jaila Children's Hospital & Mohammed Bin Rashid University | December 2021 |
| Guillaume Borque | McGill University | December 2021 |
| Corey Watson | University of Louisville | December 2021 |
| Kai Ye | Xi'an Jiaotong University | June 2022 |
| Younes Mokrab | Sidra Medicine | August 2022 |
| Nathan Sheffield |  University of Virgina | September 2022 |
| Yafei Mao | Shanghai Jiao Tong Univeristy | October 2022 |
| Mile Sikic | Genome Institute of Singapore | November 2022 |
| Jianjun Liu | Genome Institure of Singaore | November 2022 |
| Jason Chin | GeneDX | January 2023 |
| Chaochun Wei | Shanghai Jiao Tong University | January 2023 |
| Mikko Rautiainen | University of Helsinki | January 2023 |
| Alessandro Raveane | Human Technopole | May 2023
| Nicole Seranzo | Human Technopole | May 2023
| Charles Wang | Loma Linda University Center For Genomics | May 2023
| Li Song | Geisel School of Medicine at Dartmouth | June 2023 |
| Zechen Chong |University of Alabama at Birmingham | July 2023
| Xiangtao Liu | Xinjiang Medical University | June 2023
| Simon Giunta | Sapienza University of Rome | June 2023
| Floris Barthel | Translational Genomics Research Institute
| Dylan Duchen | Yale University Medical School | December 2023
| Steven Kleinstein | Yale University Medical School | December 2023
| Jia-Xing Yue | Sun Yat-sen University Cancer Center | March 2024
| Anna-Sophie Fiston-Lavier | University of Montpellier - Faculty of Sciences | August 2024
| Kateryna Makova | Penn State University | August 2024
| Mohammed Jashim Uddin | Mohammed Bin Rashid University of Medicine and Health Sciences | August 2024
| Christine Beck | Uconn Health, Jackson Lab | August 2024
| Mikhail Kolmogorov | National Cancer Institute | August 2024
| Daniel Doerr | Department of Endocrinology and Diabetology, Medical Facility, Heinrich Heine University Dusseldord | August 2024

{{< /column >}}
{{< column >}}

### Partners

{{< img src="hprcpartnerlogos.png" alt="hprc partner logos" >}}

<a class="link" href="https://www.coriell.org/" target="_blank"
                    >Coriell Institute</a
                >
                <p class="description">
                    Many of the samples used in the reference genome are made
                    available to researchers through the Coriell Institute.
                </p>
                <a
                    class="link"
                    href="https://www.ncbi.nlm.nih.gov/grc"
                    target="_blank">Genome Reference Consortium</a
                >
                <p class="description">
                    Genome Reference Consortium helps to put nucleotide
                    sequences into chromosome context for many different
                    genomes.
                </p>
                <div style="font-size: larger; font-weight: bold;">
                    Google
                </div>
                <p class="description">
                    The team at Google develops
                    <a
                        href="https://github.com/google/deepvariant"
                        target="_blank"
                        class="in_paragraph_link"
                    >
                        variant detection methods for calling with pangenomes
                    </a> and for error correction of assemblies. Google has also
                    developed <a
                        href="https://github.com/google/deepconsensus"
                        target="_blank"
                        class="in_paragraph_link"
                    >
                        sequence data correction methods that improve the output
                        of sequencing instruments
                    </a>. Google is working with HPRC members to apply that
                    technology to HPRC sequencing data and assemblies.
                </p>

{{< /column >}}
{{< /columns>}}